docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Client Sessions and Robot Networking"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How a Viam robot handles losing its connection to the internet."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet over a LAN or WAN and can no longer communicate with clients through [the Viam app](https://app.viam.com).
+
+## How does `viam-server` work if your robot loses its network connection?
+
+If the robot loses its connection to the internet, `viam-server` will timeout and end any current client [*sessions*](/program/apis/sessions/) on this robot.
+
+## How do `viam-server`'s clients work if your robot loses its network connection?
+
+If your robot running `viam-server` is no longer connected to the internet and each client session is ended, all operations will timeout automatically and operations will be halted.
+When its session times out, each client is asked to cancel any ongoing operations by the automatically-configured `SessionsClient` of your robot. 
+Any active commands will be cancelled, stopping any moving parts, and no new commands will be able to reach the robot until the connection is restored.
```

Comment:
```suggestion
## What happens to clients connected to the robot when your robot loses its network connection?

When your client cannot connect to your robot's `viam-server` instance, all client operations will timeout automatically and halt.
Any active commands will be cancelled, stopping any moving parts, and no new commands will be able to reach the robot until the connection is restored.
```

- I think a heading of "how does X work" doesn't make sense when we're describing a scenario where you could say it doesn't work - it's more like "what happens to X when"
- "When its session times out, each client is asked to cancel any ongoing operations by the automatically-configured `SessionsClient` of your robot." -> this is what happens on the server side, i.e. on the robot right? So it should move to the first part?
- giving an example of a client operation would be a good idea I think.

Reply:
Applied, thank you so much for the really good suggestions ⭐